zstacklib/zstacklib/utils/lvm.py

```
@@ -2743,3 +2748,253 @@ def check_lvmlockd_log():
         except Exception as e:
             logger.warn(str(e))
             self.failed = True
+
+
+@bash.in_bash
+def get_retry_times_for_checking_vg_lockspace():
+    r, sanlock_patch_version = bash.bash_ro("sanlock get_patch_version")
```

Ruff (0.15.5)

[warning] Unpacked variable `r` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)